packages/build/src/plugins/child/utils.js

```diff
@@ -29,19 +29,23 @@ const getBuildUtils = function(event) {
 }
 
 const getGitUtils = function() {
+  // eslint-disable-next-line node/global-require
```

Comment:
`utils` passed to plugins are lazily loaded for performance reasons. This actually speed up plugin load quite a lot.